spec/models/pacbio/run_spec.rb

```diff
@@ -4,6 +4,7 @@
 
 RSpec.describe Pacbio::Run, pacbio: true do
   let!(:version10) { create(:pacbio_smrt_link_version, name: 'v10', default: true) }
+  let!(:version12) { create(:pacbio_smrt_link_version, name: 'v12_sequel', default: true) }
```

Comment:
```suggestion
  let!(:version12) { create(:pacbio_smrt_link_version, name: 'v12_sequel_iie', default: true) }
```

Reply:
Do you want two defaults in this file? Should that error on creating `version12` as a default, when there is already a default?